<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioClipListBase</class>
 <widget class="QFrame" name="AudioClipListBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Clip List</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListView" name="m_bookmarkList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::CopyAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="txtPath">
         <property name="toolTip">
          <string>Current Path</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="m_fileList">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::CopyAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnHome">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Return to start</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRewind">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Rewind</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnForward">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Fast Forward</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnStop">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Stop playback</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPlay">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Play selected</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
